main.go

```diff
@@ -45,12 +45,10 @@ type RenameCmd struct {
 	FuzzyMatch     bool   `arg:"--fuzzy-match" help:"enable q-gram distance fuzzy matching. WARNING: You must validate by hand the output!"`
 }
 
-type MoveCmd struct {
-	UpDown
-	SrcPlanPath  string `arg:"--src-plan,required" help:"path to the SRC terraform plan"`
-	DstPlanPath  string `arg:"--dst-plan,required" help:"path to the DST terraform plan"`
-	SrcStatePath string `arg:"--src-state,required" help:"path to the SRC local state to modify"`
-	DstStatePath string `arg:"--dst-state,required" help:"path to the DST local state to modify"`
+type MoveAfterCmd struct {
+	Script string `arg:"required" help:"the migration scripts; will generate SCRIPT_up.sh and SCRIPT_down.sh"`
+	Before string `arg:"required" help:"the before root directory; will look for BEFORE.tfplan and BEFORE.tfstate"`
+	After  string `arg:"required" help:"the after root directory; will look for AFTER.tfplan and AFTER.tfstate"`
```

Comment:
Do we really look up for `BEFORE.tfstate` and `AFTER.tfstate`? In my understanding no. If they are not there the terravalet would still work, but the created up and downs script would not?

Reply:
> because the fact that Terravalet runs successfully is not enough

I agree. But why don't we add a check in the code if these files are present then? Would it be useful to add a sentence in documentation that terravalet does not perform any operation on the state files, neither remote or local state files? the generated script does.

Reply:
Right. I added PCI-2336.